source-klaviyo/source_klaviyo/schemas/events.json

```diff
@@ -4,80 +4,6 @@
   "additionalProperties": true,
   "required": ["id"],
   "properties": {
-    "type": { "type": "string" },
-    "id": { "type": "string" },
-    "datetime": { "type": "string", "format": "date-time" },
```

Comment:
`datetime` is the cursor field for the `events` stream, right? I don't think omitting it here will cause an issue, but it leaving the cursor field in shouldn't cause a problem since we're relying on it being in every document.

Reply:
@Alex-Bair There actually was a error with cursor fields being defined in the schema, which caused this [PR](https://github.com/estuary/connectors/pull/1983) to be created. I think it was more related to their schema being set as nullable, but in order to be sure i just simply remove then now